rts/Game/Camera.cpp

```diff
@@ -663,37 +642,6 @@ void CCamera::CalcFrustumLine(
 	frustumLines[index][count++] = line;
 }
 
-void CCamera::ClipFrustumLines(const float zmin, const float zmax, bool neg)
```

Comment:
Was already unused, only relevant as an alternative to minimap frustum drawing (we will migrate it to something else anyway).

Reply:
Perhaps cleaning up unused functions could be a separate commit/PR so that it would be trivial to review and merge without having to rely on the other changes.